cardano-node-chairman/test/Test/Cardano/Node/Chairman/Byron.hs

```diff
@@ -88,7 +95,9 @@ prop_spawnByronCluster = H.propertyOnce . H.workspace "chairman" $ \tempAbsPath
     H.createDirectoryIfMissing $ tempBaseAbsPath <> "/" <> socketDir
 
     H.copyFile (baseConfig <> "/topology-node-" <> si <> ".json") (tempAbsPath <> "/topology-node-" <> si <> ".json")
-    H.copyFile (baseConfig <> "/config-" <> si <> ".yaml") (tempAbsPath <> "/config-" <> si <> ".yaml")
+    H.writeFile (tempAbsPath <> "/config-" <> si <> ".yaml") . L.unlines . fmap rewriteConfiguration . L.lines =<<
```

Comment:
Wouldn't this potentially turn off all logging and only leave on: `TraceBlockchainTime: True`

Reply:
All the other logging would be as per the original configuration files: `configuration/chairman/defaults/simpleview/config-*.yaml`

The original configuration file has a few trace options turned, and those lines are preserved.

I will use this rewrite mechanism to turn on any additional logging as I need it.

Reply:
Ah I misread the code.